README.md

```diff
@@ -18,56 +18,83 @@ To install the latest released version of the library, we recommend using [conda
-    source activate jwst_devel
+    conda create -n jwst_dev --only-deps --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults python=3.6 jwst
+    source activate jwst_dev
     git clone https://github.com/STScI-JWST/jwst.git
     cd jwst
     python setup.py develop
 
-If you will not need to update the code, then you can replace the last step with
-the following command:
-
-    python setup.py install
-
-Once installed, the software can be updated to the lastest development version
-by running the following command inside the `jwst` directory: 
+Once installed, the software can be updated to the lastest development version by updating the dependencies,
+pulling the latest version of `master` from the Github repository inside the `jwst` directory: 
 
+    conda update -n jwst_dev --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults --all
     git pull origin master
+    python setup.py develop
 
 
-Contributing Code, Documentation or Feedback
---------------------------------------------
-We welcome feedback and contributions to the project. Please follow the [contributing guidelines](CONTRIBUTING.md) to submit an issue or a pull request.
+Documentation
+-------------
 
-We strive to provide a welcoming community to all of our users by abiding to the [Code of Conduct](CODE_OF_CONDUCT.md).
+Documentation (built daily from `master`) is available here:
 
-Using
------
+https://jwst-pipeline.readthedocs.io/en/latest/
 
-Documentation (latest off `master`) is available here:
 
-https://jwst-pipeline.readthedocs.io/en/latest/
+Contributions and Feedback
+--------------------------
+We welcome contributions and feedback on the project. Please follow the [contributing guidelines](CONTRIBUTING.md) to submit an issue or a pull request.
+
+We strive to provide a welcoming community to all of our users by abiding to the [Code of Conduct](CODE_OF_CONDUCT.md).
 
 Regression Tests
 ----------------
 
-Latest regression test results can be found here:
+Latest regression test results can be found here (internal STScI only):
 
 https://boyle.stsci.edu:8081/job/RT/job/JWST/
 
-Test builds start at 10am and 7pm local time every day.
+The test builds start at 10am and 7pm local Baltimore time every day on jwcalibdev.
+
+To run the regression tests on your local machine, `rsync` or `scp` the input and comparison data locally
+
+    rsync -av <username>@jwcalibdev:/data4/jwst_test_data /my/local/path/
+
+set the environment variable TEST_BIGDATA to this location
+
+    export TEST_BIGDATA=/my/local/path/jwst_test_data/
+
+and then run the tests in the repository
+
+    cd ~/path/to/jwst
+    pytest --bigdata .
+
+
+Unit Tests
+----------
+
+Unit tests can be run via `pytest`.  We recommend using `pytest-xdist` so you can run them in parallel.  Install `pytest-xdist` and run pytest in the top level of the repository
+
+    conda install pytest-xdist
+    pytest -n N
+
```

Comment:
Maybe use consistent way through the document to specify option arguments (either <> or [])